db/upgrade.php

```diff
@@ -182,6 +115,73 @@ function xmldb_tincanlaunch_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2015033100, 'tincanlaunch');
     }
 
+    if ($oldversion < 2015112702) { //New version in version.php
```

Comment:
I get that, but why did you change it in this PR? Presumably I had it set to the wrong version?

Are you sure this was the cause of your problem and not the fact that i hadn't updated intsall.xml's version number?  https://github.com/garemoko/moodle-mod_tincanlaunch/issues/93